packages/builders/plugins/jest/resolver.ts

```diff
@@ -0,0 +1,50 @@
+import { dirname } from 'path';
+import * as ts from 'typescript';
+import defaultResolver from 'jest-resolve/build/default_resolver';
+
+interface ResolveOptions {
+  rootDir: string;
+  basedir: string;
+  paths: string[];
+  moduleDirectory: string[];
+  browser: boolean;
+  extensions: string[];
+}
+
+function getCompilerSetup(rootDir: string) {
+  const tsConfigPath =
```

Comment:
why are we trying out all the tsconfig files?

Reply:
Oddly enough, `jest` does not pass the configuration options (where the user sets the tsconfig) to the `resolver`. I did not see another solution other than resolving a couple of standard tsconfig files...

Reply:
This doesn't seem right. I can see spec, test, and jest being here. But it cannot be app.